stdlib/_bisect.pyi

```diff
@@ -1,47 +1,28 @@
 import sys
-from _typeshed import SupportsRichComparison, SupportsRichComparisonT
+from _typeshed import SupportsRichComparison
 from typing import Callable, MutableSequence, Sequence, TypeVar, overload
 
-_T = TypeVar("_T")
+_T = TypeVar("_T", bound=SupportsRichComparison)
```

Comment:
Conventionally T means an unbounded TypeVar. Can you rename this to `_ SupportsRichComparisonT`?

Reply:
@JelleZijlstra Perhaps it would be better to import and re-use `SupportsRichComparisonT`?